docs/installation.md

```diff
@@ -50,6 +50,18 @@ C:\Users\myaccount> opsdroid config gen | out-file "configuration.yaml" -encodin
 C:\Users\myaccount> opsdroid start
 ```
 
+### Selecting Modules
+
+You can individually select modules to install dependencies for by specifying them
+in the square brackets. For instance, to install the redis database and webex connector
+module dependencies in addition to the opsdroid core package:
+```
+pip install opsdroid[database_redis,connector_webex]
+```
+
+`[common]` includes the matrix and slack connectors and sqlite database modules
+You can also use `[all]`, `[all_connectors]`, `[all_databases]`, `[all_parsers]` and `[test]`
```

Comment:
Can we link to the setup file here or something so users can discover all the available options?

Reply:
So add a comment block on top of setup.cfg and link to the file here?

Reply:
Not necessarily. We just need to list all the options somewhere. The `options.extras_require` section in setup.cfg lists all of the options (other than the meta ones that you already listed in the docs). So it might be useful to link there.